src/cogent3/core/sequence.py

```diff
@@ -2119,25 +2158,27 @@ def _get_forward_slice_from_forward_seqview_(self, slice_start, slice_stop, step
 
         # if -ve, it's an invalid slice
         if start < 0 or stop < 0:
-            return _zero_slice
+            return self._zero_slice
```

Comment:
change all uses to `self._zero_slice()`

Reply:
@GavinHuttley, your above comment indicates that `_zero_slice` is a property